tests/performance/live.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+MARATHON_DIR=$(pwd)
+MARATHON_PERF_TESTING_DIR=$(pwd)/marathon-perf-testing
+
+# Bring an up-to-date marathon-perf-testing environment
+[ ! -d marathon-perf-testing ] && git clone "https://${GIT_USER}:${GIT_PASS}@github.com/mesosphere/marathon-perf-testing.git"
+(cd marathon-perf-testing; git pull --rebase)
+
+# Privileged clean-up of the results folder
+docker run -i --rm \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    icharalampidis/marathon-perf-testing:latest \
+    rm -rf /marathon/results
+
+# Configuration
+DOCKER_NETWORK=testing
+
+# Get the git hash
+GIT_HASH=$(cd "$MARATHON_DIR" && git rev-parse HEAD)
+echo "- Running on GIT hash $GIT_HASH..."
+
+# Run the CI job
+docker run -i --rm \
+    --network ${DOCKER_NETWORK} \
+    --privileged \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v "$MARATHON_DIR:/marathon" \
+    -e "PARTIAL_TESTS=test-5k-apps" \
+    -e "PERF_DRIVER_ENVIRONMENT=env-ci-live.yml" \
+    -e "DATADOG_API_KEY=$DATADOG_API_KEY" \
+    -e "DATADOG_APP_KEY=$DATADOG_APP_KEY" \
```

Comment:
You haven't defined `DATADOG_API_KEY` somewhere in your environment.
It's not secret and should be set to the `dcos-perf-test-driver` app from https://app.datadoghq.com/account/settings#api

Reply:
Oh this app vs api key thing. Why do we need both? Our CI only needs the app one.

Reply:
Well, DataDog needs both 😄 The one is the _API Access Key_ the other is the accounting key.

Reply:
Ok, to be more precise, the default configuration of the library that I was using required both. Apparently you need the `API` key for submitting data and the `APP` key for reading data from DataDog.

Since the reporter only cares about sending data, you are right, we don't need an `APP` key. I am going to change the scripts on testing.